ci/common.rb

```diff
@@ -158,16 +158,16 @@ def self.for(smth, max_timeout = DEFAULT_TIMEOUT)
     task :run_tests, :flavor do |t, attr|
       flavors = attr[:flavor]
       filter = ENV['NOSE_FILTER'] || '1'
-      if flavors.include? 'default' or flavors.include? 'checks_mock'
+      if flavors.include? 'default' || flavors.include? 'checks_mock'
```

Comment:
`||` has a stronger precedence than `or`, you should add parenthesis: `flavors.include?('default')`.